sources/System/Animator/AnimatorSystem.cpp

```diff
@@ -27,6 +28,10 @@ namespace BBM
 		auto drawable = entity.get<Drawable3DComponent>().drawable.get();
 		auto &animation = entity.get<AnimationsComponent>();
 		auto anim = dynamic_cast<RAY3D::Model *>(drawable);
+		auto health = entity->tryGetComponent<HealthComponent>();
+
+		if (health && health->getHealthPoint() <= 0)
```

Comment:
You should not add random features to systems that should have nothing to do with it.